Resources/Prototypes/Entities/Objects/Weapons/Guns/pneumatic_cannon.yml

```diff
@@ -30,6 +31,7 @@
   - type: ContainerAmmoProvider
     container: storagebase
   - type: PneumaticCannon
+    throwItems: false
```

Comment:
This turns the base improvised pneumatic cannon into a gun that can fire bullets.